test/viewmodels/NationalInsuranceContributionsViewModelSpec.scala

```diff
@@ -36,16 +37,92 @@ class NationalInsuranceContributionsViewModelSpec extends SpecBase with TableDri
 
+  private val nicCannotStartStatus: Option[JourneyNameAndStatus] = Some(
+    JourneyNameAndStatus(NationalInsuranceContributions, JourneyStatus.CannotStartYet))
+
+  private val nicNotStartedStatus: Option[JourneyNameAndStatus] = Some(JourneyNameAndStatus(NationalInsuranceContributions, JourneyStatus.NotStarted))
+
+  private val nicInProgressStatus: Option[JourneyNameAndStatus] = Some(JourneyNameAndStatus(NationalInsuranceContributions, JourneyStatus.InProgress))
+
+  private val nicCompleteStatus: Option[JourneyNameAndStatus] = Some(JourneyNameAndStatus(NationalInsuranceContributions, JourneyStatus.Completed))
+
   private val testScenarios = Table(
-    ("nationalInsuranceStatuses", "userAnswers", "expected"),
-    // No statuses, no answers, defaults to cannot start yet until the saving is implemented
-    (Nil, Nil, List(expectedRow(nicUrl, NationalInsuranceContributions, CannotStartYet)))
+    ("nationalInsuranceStatus", "businessStatuses", "expected"),
+    // If Business status does not contain completed adjustment status; NIC status should be CanNotStartYet
+    (
+      None,
+      List(anEmptyTadesJourneyStatusesModel.copy(journeyStatuses = List(JourneyNameAndStatus(Adjustments, JourneyStatus.CannotStartYet)))),
+      List(expectedRow(nicUrl, NationalInsuranceContributions, CannotStartYet))),
+    (
+      nicCannotStartStatus,
+      List(aTadesJourneyStatusesModel.copy(journeyStatuses = List(JourneyNameAndStatus(Adjustments, JourneyStatus.NotStarted)))),
+      List(expectedRow(nicUrl, NationalInsuranceContributions, CannotStartYet))),
+    (
+      nicCannotStartStatus,
+      List(
+        aTadesJourneyStatusesModel.copy(
+          tradingName = Some(TradingName("TradingName1")),
+          journeyStatuses = List(JourneyNameAndStatus(Adjustments, JourneyStatus.InProgress))),
+        aTadesJourneyStatusesModel.copy(
+          tradingName = Some(TradingName("TradingName2")),
+          journeyStatuses = List(JourneyNameAndStatus(Adjustments, JourneyStatus.NotStarted)))
+      ),
+      List(expectedRow(nicUrl, NationalInsuranceContributions, CannotStartYet))),
+    (
+      nicCannotStartStatus,
+      List(
+        aTadesJourneyStatusesModel.copy(
+          tradingName = Some(TradingName("TradingName1")),
+          journeyStatuses = List(JourneyNameAndStatus(Adjustments, JourneyStatus.CannotStartYet))),
+        aTadesJourneyStatusesModel.copy(
+          tradingName = Some(TradingName("TradingName2")),
+          journeyStatuses = List(JourneyNameAndStatus(Adjustments, JourneyStatus.NotStarted))),
+        aTadesJourneyStatusesModel.copy(
+          tradingName = Some(TradingName("TradingName3")),
+          journeyStatuses = List(JourneyNameAndStatus(Adjustments, JourneyStatus.InProgress)))
+      ),
+      List(expectedRow(nicUrl, NationalInsuranceContributions, CannotStartYet))),
+
+    // If Adjustments is completed, it should show NIC status as NotStarted
+    (
```

Comment:
if adjustments in businessStatuses are ALL completed then the UI status will match the saved NICStatus or default to NotStarted.

1. None / List(Model(List(Adj, Completed))) -> NotStarted
2. NotStarted / List(Model(List(Adj, Completed)), Model(List(Adj, Completed))) -> NS
3. InProgress / List(Model(List(Adj, Completed))) -> IP
4. Completed / List(Model(List(Adj, Completed)), Model(List(Adj, Completed))) -> Completed